### DISCUSSION TITLE
New version of OpenFOAM

### DISCUSSION THREAD

**Discussion by LimousinLucas** [mdolab/dafoam#494] [General]
Hello, 

I would like to know if there is a way to put openfoam2212 or 2306 on DAFoam in order to use the SpalartAlmarasIDDES model? 

Thank you very much !

**Comment by friedenhe** [comment]
Unfortunately, we don't support 2212 or 2306.... The main limitation is that we differentiated the OpenFOAM-v1812 in a hard code manner, and we can't easily extend it to newer OF versions.
